package-lock.json

```diff
@@ -0,0 +1,564 @@
+{
+  "name": "nvm",
```

Comment:
Please remove this file; only apps should have lockfiles.

I'll gitignore it on master later today.

Reply:
Another mistake, thanks for pointing it out!